compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/MockDesugar.java

```diff
@@ -346,6 +341,7 @@ private BLangLiteral generateRHSExpr(String val) {
         return bLangLiteral;
     }
 
+    // Return: <type> MockHandler(<mockFnObj>, args)
```

Comment:
Shall we make these comments more meaningful? e.g. `// This would return the ballerina equivalent <type> MockHandler(<mockFnObj>, args)`

Reply:
Applies to other similar places

Reply:
Changed the comments in the MockDesugar class